compiler/rustc_target/src/target_features.rs

```diff
@@ -588,6 +588,14 @@ pub fn all_rust_features() -> impl Iterator<Item = (&'static str, Stability)> {
 const X86_FEATURES_FOR_CORRECT_VECTOR_ABI: &'static [(u64, &'static str)] =
     &[(128, "sse"), (256, "avx"), (512, "avx512f")];
```

Comment:
This may have to adjusted depending on how we deal with AVX10
```suggestion
const X86_FEATURES_FOR_CORRECT_VECTOR_ABI: &'static [(u64, &'static str)] =
    &[(128, "sse"), (256, "avx"), (512, "avx512f")]; // FIXME: determine if AVX10 changes this
```